code/obj/artifacts/artifactdatums.dm

```diff
@@ -24,6 +24,7 @@ ABSTRACT_TYPE(/datum/artifact/)
 
 	var/activated = 0            // Is the artifact currently switched on?
```

Comment:
every day i cri that these are not code doc comments 😿 😭

Reply:
#5177 😔